src/Controllers/organisation.controller.ts

```diff
@@ -219,13 +214,10 @@ export const deleteOrganisation = async (req: Request<DeleteOrganisationQueryPar
 
     res.status(204).end();
   } catch (e) {
```

Comment:
Why is the try catch kept here

Reply:
The try-catch is mandatory in order to respond correctly with the NotFoundError. Otherwise the response would be from the PrismaClientUnknownRequestError.